assets/lib/share-link/share-link.js

```diff
@@ -16,6 +16,17 @@
 			$element.on( 'click', function() {
 				openShareLink( networkName );
 			} );
+
+			// Add "Enter" and "Space" event only if the element has role=button attribute.
+			if ( 'button' === $element.attr( 'role' ) ) {
+				$element.on( 'keyup', ( event ) => {
+					if ( 13 === event.keyCode || 32 === event.keyCode ) {
+						event.preventDefault();
```

Comment:
I think the event.preventDefault() is redundant because the event is always on a div, and a div doesn't have a default behavior.

Reply:
Just for future coverage, if the use will be with <a> or <button>.